docs/reference/changelog.rst

```diff
@@ -16,6 +16,16 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+.. _release-2.20.0:
+
+2.20.0 (2025-MM-DD)
```

Comment:
well, this is a bugfix so it should be at least a patch release (2.19.1)
But I suggest we roll this back into 2.19.0